docs/Concepts/Pruning.md

```diff
@@ -1,25 +1,61 @@
 ---
-description: Pruning
+description: Full and archive node types
 ---
 
-# Pruning
+# Full and archive node types
+
+Besu supports two node types, commonly referred to as _full nodes_ and _archive nodes_.
+
+Full nodes have just the current state of the blockchain so cannot serve the network with all data
```

Comment:
I would remove the `just` to reflect that they can have more than the current state.

Reply:
Done.